src/fsharp/CompileOps.fs

```diff
@@ -2000,11 +2000,12 @@ let ResolveFileUsingPaths(paths, m, name) =
         let searchMessage = String.concat "\n " paths
         raise (FileNameNotResolved(name, searchMessage, m))            
 
-let GetWarningNumber(m, s:string) =
+let GetWarningNumber(s:string) =
     try 
-        Some (int32 s)
-    with err -> 
-        warning(Error(FSComp.SR.buildInvalidWarningNumber(s), m))
```

Comment:
It now also no longer warns for invalid integers, or an invalid FSxxx, right?

Reply:
@0x53a Correct ... There is not really the notion of an invalid FSxxx ... either the value was an integer or it failed.  We did not check that the integer mapped to an actual error or warning number.

Reply:
> We did not check that the integer mapped to an actual error or warning number.

Ah, that's what I missed. Thanks.